sparkles/tests/test_review.py

```diff
@@ -117,8 +114,26 @@ def test_probs_weak_reference():
     assert aca.guides is not acar.guides
     assert aca.acqs is not acar.acqs
 
-    del aca
-    gc.collect()
 
-    for probs in acar.acqs.cand_acqs['probs']:
-        assert probs.acqs() is acar.acqs
+def test_roll_options_with_include_ids():
+    """
+    Test case from James that was breaking code due to a roll option that puts
+    a force_include star outside the FOV.
+
+    """
+    kwargs = {'obsid': 48397.0,
+              'att': [0.43437703, -0.47822201, -0.68470554, 0.33734053],
+              'date': '2019:053:04:05:33.004', 'detector': 'ACIS-S',
+              'dither_acq': (7.9992, 2.0016), 'dither_guide': (7.9992, 2.0016),
+              'man_angle': 131.2011858838081, 'n_acq': 8, 'n_fid': 0, 'n_guide': 8,
+              'sim_offset': 0.0, 'focus_offset': 0.0, 't_ccd_acq': -12.157792574498563,
+              't_ccd_guide': -12.17,
+              'include_ids_acq': np.array(
+                  [8.13042280e+08, 8.13040960e+08, 8.13044168e+08, 8.12911064e+08,
+                   8.12920176e+08, 8.12913936e+08, 8.13043216e+08, 8.13045352e+08]),
+              'include_halfws_acq': np.array(
+                  [160., 160., 160., 160., 160., 160., 120., 60.])}
+
+    aca = get_aca_catalog(**kwargs)
```

Comment:
I suppose the test does not need to raise_exc=True if we are moving that to the default.

Reply:
`acar = aca.get_review_table()` will fail (with an exception) if `aca` is not a well-formed catalog with all the necessary columns.  So this is a valid test with or without the default change.